src/test/java/org/sagebionetworks/web/unitclient/widget/profile/UserProfileEditorWidgetImplTest.java

```diff
@@ -214,7 +214,6 @@ private void verifyConfigure(int nCalls) {
     verify(mockView, times(nCalls)).setLocation(profile.getLocation());
     verify(mockView, times(nCalls)).setLink(profile.getUrl());
     verify(mockView, times(nCalls)).setBio(profile.getSummary());
-    verify(mockView, times(nCalls)).setEmail(EMAIL2);
```

Comment:
verifyConfigure() no longer checks for view.setEmail(), we do this in the individual test scenarios.